demos/python_demos/instance_segmentation_demo/README.md

```diff
@@ -57,12 +57,19 @@ Options:
   -l "<absolute_path>", --cpu_extension "<absolute_path>"
                         Required for CPU custom layers. Absolute path to a
                         shared library with the kernels implementation.
+  --delay "<num>"       Optional. Interval in millisecond to wait for key
```

Comment:
`Interval in millisecond to wait for key` --> `Interval in milliseconds of waiting for a key to be`

Reply:
Fixed. Thanks.